tests/test_balancing_agent.py

```diff
@@ -18,6 +18,7 @@
 import pytest
 
 import pendulum
+from datetime import datetime
```

Comment:
AS we try to use pendulum solely, please use pendulum.now() for the "now" Thanks!